<?xml version='1.0' encoding='UTF-8'?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSoC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayoutCentralWidget">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayoutToolList">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutBusEditor">
        <item>
         <widget class="QToolButton" name="toolButtonBusEditor">
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutBusEditor">
          <item>
           <widget class="QLabel" name="labelBusEditor">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Bus Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBusEditorDetail">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Edit the project bus definition</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutSchematicEditor">
        <item>
         <widget class="QToolButton" name="toolButtonSchematicEditor">
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSchematicEditor">
          <item>
           <widget class="QLabel" name="labelSchematicEditor">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Schematic Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSchematicEditorDetail">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Edit the project schematic</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutModuleEditor">
        <item>
         <widget class="QToolButton" name="toolButtonModuleEditor">
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutModuleEditor">
          <item>
           <widget class="QLabel" name="labelModuleEditor">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Module Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelModuleEditorDetail">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Edit golbal and/or project schematic modules</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacerToolList">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject" />
    <addaction name="actionOpenProject" />
    <addaction name="separator" />
    <addaction name="actionCloseProject" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRefresh" />
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBusEditor" />
    <addaction name="actionSchematicEditor" />
    <addaction name="actionModuleEditor" />
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuTools" />
   <addaction name="menuPreferences" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="dockWidgetProjectFile">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>109</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string extracomment="List of project files">Project Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTreeView" name="treeViewProjectFile" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject" />
   <addaction name="actionOpenProject" />
   <addaction name="actionCloseProject" />
   <addaction name="separator" />
   <addaction name="actionRefresh" />
   <addaction name="separator" />
   <addaction name="actionOpenProjectInFileExplorer" />
  </widget>
  <action name="actionNewProject">
   <property name="icon">
    <iconset theme="document-new" />
   </property>
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="toolTip">
    <string>Create new blank project (CTRL+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset theme="document-open" />
   </property>
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open an existing project (CTRL+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="icon">
    <iconset theme="document-close" />
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
   <property name="toolTip">
    <string>Close the current project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="window-close" />
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit qsoc (CTRL+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh" />
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh project files (F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionOpenProjectInFileExplorer">
   <property name="icon">
    <iconset theme="document-open-in-explorer" />
   </property>
   <property name="text">
    <string>Open In File Explorer</string>
   </property>
   <property name="toolTip">
    <string>Open In File Explorer</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionBusEditor">
   <property name="icon">
    <iconset theme="applications-bus" />
   </property>
   <property name="text">
    <string>&amp;Bus Editor</string>
   </property>
   <property name="toolTip">
    <string>Edit project bus definitions (CTRL+B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionModuleEditor">
   <property name="icon">
    <iconset theme="applications-module" />
   </property>
   <property name="text">
    <string>Symbo&amp;l Editor</string>
   </property>
   <property name="toolTip">
    <string>Edit schematic modules (CTRL+L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSchematicEditor">
   <property name="icon">
    <iconset theme="applications-schematic" />
   </property>
   <property name="text">
    <string>Schematic &amp;Editor</string>
   </property>
   <property name="toolTip">
    <string>Edit schematic (Ctrl+E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources />
 <connections />
</ui>